<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenGL 纹理环绕方式</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="OpenGLView" name="openGLWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRepeat"/>
   <addaction name="actionMirroredRepeat"/>
   <addaction name="actionClampToEdge"/>
   <addaction name="actionClampToBorder"/>
  </widget>
  <action name="actionRepeat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Repeat</string>
   </property>
  </action>
  <action name="actionMirroredRepeat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>MirroredRepeat</string>
   </property>
  </action>
  <action name="actionClampToEdge">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ClampToEdge</string>
   </property>
  </action>
  <action name="actionClampToBorder">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ClampToBorder</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OpenGLView</class>
   <extends>QOpenGLWidget</extends>
   <header>openglview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
